index.html

```diff
@@ -8,8 +8,7 @@
         <meta name="description" content="Veteran-founded and led, Operation Code is a 501(c)(3) non-profit whose mission is to aid military, vets and their families learn coding and web technologies.">
         <meta name="keywords" content="operation code, veterans, software development, learn to code, keep coding, veteran coders, senate veterans, house veterans, gi bill, new gi bill, operationcode.org, #operationcode, military vets, vets">
         <meta name="author" content="Veteran Coders, for Veterans Coders">
-        <meta property="og:image" content="https://static1.squarespace.com/static/586d87de725e2585df6e8448/t/5870110d6b8f5b61fdd8ee05/1483739408996/?format=500w" />
-
+        <meta property="og:image" content="src/images/logos/small-stacked-logo-blue.png" />
```

Comment:
I just copied the relative path, I'm not sure how or where the image URLs are being served. @kylemh